internal/api/helpers.go

```diff
@@ -19,6 +19,12 @@ func sendJSON(w http.ResponseWriter, status int, obj interface{}) error {
 	return shared.SendJSON(w, status, obj)
 }
 
+func sendTokenJSON(w http.ResponseWriter, status int, obj interface{}) error {
+	w.Header().Set("Cache-Control", "no-store, no-cache, must-revalidate")
+	w.Header().Set("Pragma", "no-cache")
```

Comment:
These apply just in general to all Auth requests, so maybe it can be added as middleware?